crates/sui-node/src/lib.rs

```diff
@@ -1593,9 +1586,20 @@ impl SuiNode {
                         .get_signed_effects_digest(tx.digest())
                         .expect("db error")
                     {
-                        pending_consensus_certificates.push((tx, fx_digest));
+                        pending_consensus_certificates.push((
+                            Schedulable::Transaction(tx),
+                            ExecutionEnv {
```

Comment:
same question here re builder vs not, and they're right next to each other

applies in more places but will stop commenting individually